x/storage/types/codec.go

```diff
@@ -33,6 +34,16 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgDeleteBucket{},
 	)
+	registry.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
Can you also help to register `MsgDiscontinueBucket` and `MsgDiscontinueObject`? Thanks

Reply:
It has already registered MsgDiscontinueBucket & MsgDiscontinueObject.